### ISSUE TITLE
support resque 2.0?

### ISSUE THREAD

**Issue by jrochkind** [nevans/resque-pool#170]
A resque 2.0 was finally released on [November 6 2018](https://rubygems.org/gems/resque/versions/2.0.0)

The latest release of resque-pool is 0.6.0, with a gemspec that says resque `~> 1.22` -- so 1.x, not 2.0. 

It's possible resque-pool would just work with resque 2.0 if the gemspec allowed it, there aren't many [changes listed in the resque changes file](https://github.com/resque/resque/blob/master/HISTORY.md#200-2018-11-06). 

Resque 2.0 does support redis 4, and is presumably needed in order to support redis 4.

Is this project still supported? I'm a bit confused as to the whole resque eco-system these days!

**Comment by jesperronn** [comment]
This can be closed after #171 is merged. (but we are missing the release...will create separate issue for that)

**Comment by nevans** [comment]
Please try `v0.7.0`, which includes #171. Let me know how it works for you, thanks!
